src/apollo.ts

```diff
@@ -1,15 +1,54 @@
 import { ApolloServer, gql } from 'apollo-server';
 import { ApolloServerPluginLandingPageLocalDefault } from 'apollo-server-core';
 
+interface UserInput {
+  id?: number;
+  name: string;
+  email: string;
+  password: string;
+  birthdate: string;
+}
+
 const typeDefs = gql`
   type Query {
-    hello: String
+    users: [User]
+  }
+
+  type Mutation {
+    createUser(name: String!, email: String!, password: String!, birthdate: String!): User!
```

Comment:
ao invés de utilizar um argumento pra cada propriedade do user, poderia criar um input type do graphql e passar apenas esse argumento. https://graphql.org/graphql-js/mutations-and-input-types/

Reply:
opa, vi que mudou em outro PR 😅